lib/analyzer/apt-analyzer.ts

```diff
@@ -0,0 +1,115 @@
+import subProcess from '../sub-process';
+
+export {
+  analyze,
+};
+
+async function analyze(targetImage) {
+  const dpkgFile = await dockerCat(targetImage, '/var/lib/dpkg/status');
+  const pkgs = parseDpkgFile(dpkgFile);
+
+  const extFile = await dockerCat(targetImage, '/var/lib/apt/extended_states');
+  if (extFile) {
+    setAutoInstalledPackages(extFile, pkgs);
+  }
+
+  return {
+    Image: targetImage,
+    AnalyzeType: 'Apt',
+    Analysis: pkgs,
+  };
+}
+
+async function dockerCat(targetImage, file): Promise<string> {
```

Comment:
I think you sould use this function in all of the other places you're cat-ing files from the img